aspnetcore/toc.yml

```diff
@@ -604,9 +604,9 @@
       uid: web-api/advanced/conventions
     - name: Handle errors
       uid: web-api/handle-errors
-    - name: HTTP REPL
+    - name: Test with a REPL
```

Comment:
I think "Test with HttpRepl" works better but if there are reasons why we can't, I get it.